src/gui/resources/translations/de.ts

```diff
@@ -0,0 +1,2749 @@
+    </message>
+    <message>
+        <location filename="../../kbbindwidget.cpp" line="102"/>
+        <source>&lt;center&gt;Reset this %1 to default?&lt;/center&gt;</source>
+        <translation>&lt;center&gt;Diese %1 auf Standard zurücksetzen?&lt;/center&gt;</translation>
+    </message>
+    <message>
+        <location filename="../../kbbindwidget.cpp" line="104"/>
+        <source>&lt;center&gt;Reset %1 %2s to default?&lt;/center&gt;</source>
+        <translation>&lt;center&gt;%1 %2n auf Standard zurücksetzen?&lt;/center&gt;</translation>
+    </message>
+    <message>
+        <location filename="../../kbbindwidget.cpp" line="105"/>
+        <source>Confirm action</source>
+        <translation>Aktion bestätigen</translation>
+    </message>
+    <message>
+        <location filename="../../kbbindwidget.cpp" line="97"/>
+        <location filename="../../kbbindwidget.cpp" line="115"/>
+        <source>button</source>
+        <translation>Maustaste</translation>
+    </message>
+    <message>
+        <location filename="../../kbbindwidget.cpp" line="97"/>
+        <location filename="../../kbbindwidget.cpp" line="115"/>
+        <source>key</source>
+        <translation>Taste</translation>
+    </message>
+    <message>
+        <location filename="../../kbbindwidget.cpp" line="116"/>
+        <source>s</source>
+        <translation>n</translation>
+    </message>
+    <message>
+        <location filename="../../kbbindwidget.cpp" line="120"/>
+        <source>all %1s</source>
+        <translation>Alle %1n</translation>
+    </message>
+    <message>
+        <location filename="../../kbbindwidget.cpp" line="122"/>
+        <source>Copy binding for </source>
+        <translation>Zuordnung kopieren für </translation>
+    </message>
+    <message>
+        <location filename="../../kbbindwidget.cpp" line="122"/>
+        <source> to:</source>
+        <translation> nach:</translation>
+    </message>
+</context>
+<context>
+    <name>KbLightWidget</name>
+    <message>
+        <location filename="../../kblightwidget.ui" line="52"/>
+        <source>Colors</source>
+        <translation>Farben</translation>
+    </message>
+    <message>
+        <location filename="../../kblightwidget.ui" line="71"/>
+        <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If enabled, the same colors will be displayed here as on the keyboard. If disabled, the base colors will be shown.&lt;/p&gt;&lt;p&gt;Note: The animation preview will be disabled when the window loses focus to preserve computation resources.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
+        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If enabled, the same colours will be displayed here as on the keyboard. If disabled, the base colors will be shown.&lt;/p&gt;&lt;p&gt;Note: The animation preview will be disabled when the window loses focus to preserve computation resources.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
```

Comment:
This must have happened because it was already marked as finished in the en_GB.ts I used. 

I’d add this:

> Wenn aktiviert, werden hier die gleichen Farben angezeigt wie auf der Tastatur. Wenn deaktiviert, werden die Grundfarben angezeigt.</p><p>Hinweis: Die Animationsvorschau wird deaktiviert, wenn das Fenster den Fokus verliert, um Systemressourcen zu sparen.

![Tooltip - Animiert anzeigen](https://user-images.githubusercontent.com/30179385/128515155-3931971e-0358-4a8a-8ef2-2e8786a4888c.png)